src/components/ExpansionPanel/Body/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Body = ({ children, ...passedProps }) => (
+  <div {...passedProps}>{children}</div>
+);
+
+Body.propTypes = {
+  /**
+   * The content to render.
+   */
+  children: PropTypes.oneOfType([
```

Comment:
Repeat mention of the shared `propTypes.js` file that could share this

Reply:
I think that `PropTypes.node` is what I was looking for